### PR TITLE
DM-13260: constructCalibs: check for full visit/ccd set

### PR DESCRIPTION
**Pull request by PaulPrice**
We don't want to combine a different set of inputs for different CCDs
(some operations implicitly assume that, e.g., flat-fields), so check
the inputs and report if there's a problem.

### CODE REVIEW

File: python/lsst/pipe/drivers/constructCalibs.py [hunk 1]
```diff
@@ -194,7 +195,7 @@ def dictToTuple(dict_, keys):
     return tuple(dict_[k] for k in keys)
 
 
-def getCcdIdListFromExposures(expRefList, level="sensor", ccdKeys=["ccd"]):
+def getCcdIdListFromExposures(expRefList, level="sensor", ccdKeys=["ccd"], visitKeys=["visit"]):
```
Comment (rearmstr) [inline python/lsst/pipe/drivers/constructCalibs.py]: Why is this added as an argument?

**Review by rearmstr — APPROVED** [review]

File: python/lsst/pipe/drivers/constructCalibs.py (hunk 1 above)
Comment (PaulPrice) [reply python/lsst/pipe/drivers/constructCalibs.py]: Good catch. It's vestigial, from before I made `checkCcdIdLists` a method instead of a free function.
